src/models/Settings.js

```diff
@@ -139,8 +140,10 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
 
       //Registration
       'registration.click': 'function',
-      'registration.clientId': 'string'
+      'registration.clientId': 'string',
 
+      //Consent
+      'consent': 'object'
```

Comment:
shouldn't this be ```'consent.cancel': 'function'``` ?
We only care about that function as far as I can remember...